src/main.rs

```diff
@@ -135,12 +157,13 @@ async fn main() -> anyhow::Result<()> {
                 None
             } else {
                 // Initialize the client to get the fee params
-                let (_, client) = ForkDetails::fork_network_and_client("mainnet")
-                    .map_err(|e| anyhow!("Failed to initialize client: {:?}", e))?;
+                let (_, client) = ForkDetails::fork_network_and_client("mainnet").map_err(|e| {
+                    generic_error_msg(format!("Failed to initialize client: {e:?}"))
```

Comment:
could genereric_error_msg be a 'macro' -- so that I can direclty replace anyhow! (and dont' have to run the 'format' inside?

Reply:
Sure! For the PoC we have deliberately tried to not have any macro magic to keep it simple.